wgpu-core/src/validation.rs

```diff
@@ -1179,24 +1179,14 @@ impl Interface {
         Ok(outputs)
     }
 
-    pub fn has_dual_source_blending_entry_point(&self) -> bool {
```

Comment:
Removing this means that we will not validate any entry points that are not referenced by the pipeline. I thought you wanted that, but it is ok by me to not do it.

Reply:
Since the entry point won't be used by the pipeline, we shouldn't take it into account for validation.